Examples/src/main/java/com/aspose/imaging/cloud/reverseimagesearch/FindDuplicateImages.java

```diff
@@ -0,0 +1,99 @@
+import com.aspose.imaging.cloud.sdk.model.ImageDuplicates;
+import com.aspose.imaging.cloud.sdk.model.ImageDuplicatesSet;
+import com.aspose.imaging.cloud.sdk.model.SearchResult;
+import com.aspose.imaging.cloud.sdk.model.requests.FindImageDuplicatesRequest;
+import com.aspose.imaging.cloud.sdk.model.requests.UploadFileRequest;
+import org.apache.commons.io.IOUtils;
+
+import java.io.File;
+import java.io.FileInputStream;
+
+public class FindDuplicateImages extends ImagingAIBase {
+    /*
+     * Find duplicate images.
+     */
+    public void findImageDuplicates() {
+        // Create ImagingApi instance
+        ImagingApi imagingApi = new ImagingApi(APP_KEY, APP_SID);
+
+        // Create new search context
+        String searchContextId = createImageSearch();
+
+        String comparableImage = "ComparableImage.jpg";
+        String comparingImageSimilarLess15 = "ComparingImageSimilar15.jpg";
+        String comparingImageSimilarMore75 = "ComparingImageSimilar75.jpg";
+
+        try {
+            // Upload images to Cloud Storage
+            String[] images = {comparableImage, comparingImageSimilarLess15, comparingImageSimilarMore75};
+            for (String imageName : images) {
+                File inputFile = new File(DATA_PATH + imageName);
+                FileInputStream localInputImageStream = new FileInputStream(inputFile);
+
+                try {
+                    byte[] localInputImage = IOUtils.toByteArray(localInputImageStream);
+
+                    UploadFileRequest uploadFileRequest = new UploadFileRequest(imageName, localInputImage, null);
+                    FilesUploadResult result = imagingApi.uploadFile(uploadFileRequest);
+                } finally {
+                    if (localInputImageStream != null) {
+                        localInputImageStream.close();
+                    }
+                }
+            }
+
+            createImageFeatures(comparableImage, false, searchContextId);
+            createImageFeatures(comparingImageSimilarLess15, false, searchContextId);
+            createImageFeatures(comparingImageSimilarMore75, false, searchContextId);
+
+            Double similarityThreshold = 80.0; // The similarity threshold
+            String folder = null; // Path to input files
+            String storage = null; // We are using default Cloud Storage
+
+            ImageDuplicatesSet result = imagingApi.findImageDuplicates(
+                    new FindImageDuplicatesRequest(searchContextId, similarityThreshold, folder, storage));
+
+            for (ImageDuplicates duplicates : result.getDuplicates())
+            {
+                System.out.println("Duplicates:");
+                for (SearchResult duplicate : duplicates.getDuplicateImages())
+                {
+                    System.out.println("ImageName: " + duplicate.getImageId() +
+                            ", Similarity: " + duplicate.getSimilarity());
+                }
+            }
```

Comment:
@sohail-aspose Please, add 
```java
deleteImageSearch(searchContextId);
```
as in other use-cases. Thanks.

Reply:
Done.